packages/wrangler/src/inspect.ts

```diff
@@ -315,7 +315,7 @@ export default function useInspector(props: InspectorProps) {
     };
   }, [
     props.inspectorUrl,
-    retryRemoteWebSocketConnectionSigil,
+    retryRemoteWebSocketConnectionSigil, // Do NOT remove this sigil from the dependency list.
```

Comment:
In a few years time, when you come back to look at this code, I wonder if you'll be thinking, "But why should I not remove this?"